src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> reports = new HashMap<>();
```

Comment:
The access modifier for `reports` should be private to follow the encapsulation principle. It's a good practice to expose the data through methods rather than directly allowing public access to class fields.